### ISSUE TITLE
Very well done....BUT...(considerations)

### ISSUE THREAD

**Issue by Zibri** [SandboxEscaper/randomrepo#5]
Releasing this publicly, you did not accomplish anything important:

1) bad people will take advantage of it on unpatched systems.
2) you didn't get any money out of this.
3) you allow microsoft to obtain a full bug report and vulnerability FOR FREE (as theya re not rich enough)

And for what? Few lines on security websites most non technical people don't even know of (and that will make money anyway by advertising on their articles about his and other things).

In other words: you made many people make money without getting anything. I hope you are rich by your own, dude, but anyway this was IMHO a veru bad move.

**Comment by j4xx88** [comment]
I bet MS wouldn't have paid for this, too. 

Now MS is forced handle the bug.  You did the right thing! :)

**Comment by 20esaua** [comment]
@LaurentBouchard Agreed. But @j4xx88, this is actually a $30,000 bug: https://zerodium.com/images/zerodium_prices.png

**Comment by LaurentBouchard** [comment]
@jesuiscamille That seems like an awesome nonprofit organisation idea. It would be really nice to see a sort of "Hack4School" company arise in the next few years.

**Comment by jesuiscamille** [comment]
@LaurentBouchard yeah !
